apps/frontend/src/shared-research/__tests__/ResearchOutput.test.tsx

```diff
@@ -69,64 +69,206 @@ const renderComponent = async () => {
     mockGetResearchOutput.mockResolvedValue({
       ...createResearchOutputResponse(),
       type: 'Protocol',
-      tags: ['Example Tag'],
-      title: 'Not-Grant-Document title!',
+      title: 'Protocol title!',
+      description: 'Protocol contents',
+      accessInstructions: 'instructions',
+      asapFunded: true,
+      sharingStatus: 'Public',
+      contactEmails: ['pmmail'],
     });
-    const { getByRole, getByText } = await renderComponent();
-    expect(getByText(/Example Tag/i)).toBeVisible();
-    expect(getByRole('heading', { level: 1 }).textContent).toEqual(
-      'Not-Grant-Document title!',
+    const { getByText } = await renderComponent();
+    expect(getByText(/protocol title/i)).toBeVisible();
+    expect(getByText(/protocol contents/i)).toBeVisible();
+    expect(getByText(/protocol contents/i)).toBeVisible();
+    expect(getByText(/description/i)).toBeVisible();
+    expect(getByText(/access instructions/i)).toBeVisible();
+    expect(getByText(/additional information/i)).toBeVisible();
+    expect(getByText(/contact pm/i).closest('a')).toHaveAttribute(
+      'href',
+      expect.stringMatching(/pmmail/i),
     );
   });
-});
 
-it('renders the 404 page for a missing research output', async () => {
-  mockGetResearchOutput.mockResolvedValue(undefined);
-  const { getByText } = await renderComponent();
-  expect(getByText(/sorry.+page/i)).toBeVisible();
+  it('renders the Lab resource type', async () => {
+    mockGetResearchOutput.mockResolvedValue({
+      ...createResearchOutputResponse(),
+      type: 'Lab Resource',
+      title: 'Lab Resource title!',
+      description: 'Lab Resource contents',
+      accessInstructions: 'instructions',
+      asapFunded: true,
+      sharingStatus: 'Public',
+      contactEmails: ['pmmail'],
+    });
+    const { getByText } = await renderComponent();
+    expect(getByText(/lab resource title/i)).toBeVisible();
+    expect(getByText(/lab resource contents/i)).toBeVisible();
+    expect(getByText(/lab resource contents/i)).toBeVisible();
+    expect(getByText(/description/i)).toBeVisible();
+    expect(getByText(/access instructions/i)).toBeVisible();
+    expect(getByText(/additional information/i)).toBeVisible();
+    expect(getByText(/contact pm/i).closest('a')).toHaveAttribute(
+      'href',
+      expect.stringMatching(/pmmail/i),
+    );
+  });
+
+  it('renders the Bioinformatics type', async () => {
+    mockGetResearchOutput.mockResolvedValue({
+      ...createResearchOutputResponse(),
+      type: 'Bioinformatics',
+      title: 'Bioinformatics title!',
+      description: 'Bioinformatics contents',
+      accessInstructions: 'instructions',
+      asapFunded: true,
+      sharingStatus: 'Public',
+      contactEmails: ['pmmail'],
+    });
+    const { getByText } = await renderComponent();
+    expect(getByText(/bioinformatics title/i)).toBeVisible();
+    expect(getByText(/bioinformatics contents/i)).toBeVisible();
```

Comment:
Can we not do this as a it.each? This is 5 duplicated tests which essentially test one prop difference.